contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,299 @@
+            "foreign_table": foreign_table,
+            "foreign_table_key_column": foreign_table_key_column,
+        }
+
+        return [
+            RenderedStringTemplateContent(
+                content_block_type="string_template",
+                string_template={
+                    "template": template_str,
+                    "params": params,
+                    "styling": styling,
+                },
+            )
+        ]
+
+    @classmethod
+    @override
+    @renderer(renderer_type="renderer.diagnostic.unexpected_table")
+    def _diagnostic_unexpected_table_renderer(
+        cls,
+        configuration: ExpectationConfiguration = None,
+        result: ExpectationValidationResult = None,
+        runtime_configuration: Optional[dict] = None,
+        **kwargs,
+    ):
+        if result is None:
+            return None
+
+        result_dict: Optional[dict] = result.result
+
+        if result_dict is None:
+            return None
+
+        unexpected_index_list: Optional[List[dict]] = result_dict.get(
+            "unexpected_index_list"
+        )
+        # Don't render table if we don't have unexpected_values
+        if not unexpected_index_list:
+            return None
+
+        unexpected_index_df: pd.DataFrame = pd.DataFrame(
+            unexpected_index_list, dtype="string"
+        )
+
+        # extract column name from unexpected values
+        column_name: str = list(unexpected_index_list[0].keys())[0].upper()
+        header_row = [f"Missing Values for {column_name} Column"]
+
+        row_list = []
+        for index, row in unexpected_index_df.iterrows():
+            unexpected_value = row
+            row_list.append(unexpected_value)
+
+        unexpected_table_content_block = RenderedTableContent(
+            **{  # type: ignore[arg-type]
+                "content_block_type": "table",
+                "table": row_list,
+                "header_row": header_row,
+                "styling": {
+                    "body": {"classes": ["table-bordered", "table-sm", "mt-3"]}
+                },
+            }
+        )
+        return [unexpected_table_content_block]
+
+    def _validate(
+        self,
+        configuration: ExpectationConfiguration,
+        metrics: dict,
+        runtime_configuration: Optional[dict] = None,
+        execution_engine: Optional[ExecutionEngine] = None,
+    ) -> Union[ExpectationValidationResult, dict]:
+        unexpected_values = metrics.get("query.template_values")
+        final_value = len(unexpected_values)
+
+        return ExpectationValidationResult(
+            success=(final_value == 0),
+            result={
+                "observed_value": f"{final_value} missing value{'s' if final_value != 1 else ''}.",
+                "unexpected_index_list": unexpected_values,
+            },
+        )
+
+
+if __name__ == "__main__":
+    ExpectColumnValuesToBePresentInAnotherTable().print_diagnostic_checklist()
```

Comment:
This will break because you need explicit args to be passed in to the constructor now (no optional configuration) - I'd like to make `print_diagnostic_checklist` a classmethod but I don't have the bandwidth to do that refactor right now

Reply:
I'd lean towards deleting this whole `if __name__` block or just commenting it out

Reply:
Ok to remove.